src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/ConfigurationEndpoints.cs

```diff
@@ -13,6 +16,14 @@ public void Register(IEndpointRouteBuilder endpointRouteBuilder)
       var appConfig = configuration.Get<ApplicationConfiguration>();
       return TypedResults.Ok(appConfig);
     }).WithOpenApi("Returns the UI initial configuration", string.Empty, "configuration_get");
+
+    endpointRouteBuilder.MapGet("/api/provincelist", async (HttpContext ctx, IMediator messageBus, IMapper mapper, CancellationToken ct) =>
+    {
+      var results = await messageBus.Send(new ProvincesQuery(), ct);
+      return TypedResults.Ok(mapper.Map<IEnumerable<Province>>(results.Items));
+    }).WithOpenApi("Handles province queries", string.Empty, "province_get")
+      .RequireAuthorization()
```

Comment:
Not sure if you want to protect this endpoint. Also, it is worth enabling output caching and/or cache the list in the distributed cache.